apis/milvus.io/v1beta1/milvus_types.go

```diff
@@ -120,7 +118,8 @@ func (ms MilvusSpec) GetMilvusVersionByImage() (*semver.Version, error) {
 	if len(splited) != 2 {
 		return nil, errors.Errorf("unknown version of image[%s]", splited[0])
 	}
-	return util.GetSemanticVersion(splited[1])
+	v, err := semver.ParseTolerant(splited[1])
+	return &v, err
```

Comment:
@haorenfsa, what do you think of changing the return signature of this function to return the value instead of the pointer?

```suggestion
	return semver.ParseTolerant(splited[1])
```

Reply:
@SuperQ Both are fine to me. I used the pointer because `"github.com/coreos/go-semver/semver"`.NewVersion returns a pointer

Reply:
Updated